Pages/Page 1_Intro TOC.md

```diff
@@ -46,23 +42,45 @@ _insert image of app_
 * Firefox with Firebug
 * The Heroku Toolbelt
 
-Everything should be set up the night before during our install-fest.
-Please ensure you have everything working _before_ you show up for
-ClojureBridge on Saturday.
+_These should have all been installed at the Friday night Install Fest_
+
+
+## Readme
+This is the standard structure of most coding projects. You will have a Readme file that will provide a basic overview of the project and the programs you need to open the project. 
 
-First, we're going to verify that everything is set up correctly.
-We'll do this by entering some commands in the terminal.
+### App
 
-When we demonstrate terminal commands, we'll show a snippet from our
-terminal.  Our prompt is the dollar sign; everything after the dollar
-sign is the command we typed.  The result of the command is printed on
-the following lines.  Sometimes we'll have a sequence of commands in
-the same snippet — look for the dollar signs!  The terminal you're
-using will probably have different prompts but the commands should
-work the same.
+In this curriculum we will create a web app, the Chatter app, that displays posted messages. 
 
+### Prerequisites
+
+You will need [Leiningen][] 2.0.0 or above installed. 
+
+
+[leiningen]: https://github.com/technomancy/leiningen
+
+### Running The Server & Entering Commands in the terminal
+
+#### Server
+Confirm your server is running properly. To start a web server for the application, enter the following command in the terminal:
+
+    lein ring server
```

Comment:
I double checked and this won't work here.  The student would get an error.

The version check is intact in the "Entering Commands" section.  (Typo in "Comands")  Maybe we could just move the server section later in the tutorial?